src/api-reference/invoice/purchase-request-endpoints.markdown

```diff
@@ -3,26 +3,33 @@ title: Purchase Request
 layout: reference
 ---
 
-{% include prerelease.html %}
-
 # Purchase Request
-
 * [Create a new purchase request](#post)
 * [Get details of purchase request](#get)
 * [Schema](#create_purchase_request_schema)
-* [Response schema](#create_purchase_request_schema-response)
+  * [Create Purchase Request Schema](#create_purchase_request_schema)
+  * [Create Purchase Response Schema](#create_purchase_request-response-schema)
+  * [Get Purchase Request Response Schema](#get_purchase_request_response_schema)
+  * [Error](#error)
 * [Error codes](#error-codes)
 
-## Version
 
-4.0
+## Version
+4.0<br>
+**PRE-RELEASE:** The API is in Pre-Release and that only authorized early adopters should be using the API. Please contact <a href="mailto:charles.muellner@sap.com">charles.muellner@sap.com</a> for more information.
 
 ## <a name="post"></a>Create a new purchase request  
 
     POST  /purchaserequest/v4/purchaserequests
 
 Create a Purchase Request based on provided header and line item details. If the request is valid it returns back a unique identifier to look at purchase request details and creates a purchase request in back ground.
 
+### <a name="8"></a>Scopes 
+
+|Name | Description
```

Comment:
Please see the proper table formatting within the documentation template at https://github.com/concur/developer.concur.com/blob/preview/src/document-template.markdown

Reply:
This applies to all new tables added.

Reply:
I did not understand what I am missing in table formatting. I did compare with resource you have provided, but I missing some thing.